docs-api/frontity-packages/collections-packages/hooks/infinite-scroll-hooks.md

```diff
@@ -90,7 +35,7 @@ The hook receives options to set a limit of pages shown automatically, to disabl
 
 Apart from that list, it returns a set of boolean values to know if the next pages is being fetched, if the limit has been reached or if the next page returned an error, and a function to fetch the next page manually.
```

Comment:
Change:
"Apart from that list, it returns a set of boolean values to know if the next pages is being fetched, if the limit has been reached or if the next page returned an error, and a function to fetch the next page manually."
to:
"In addition to the above, the hook returns a set of boolean values that indicate if the next page is being fetched, if the limit has been reached, or if the next page returned an error, and a function that allows the next page  to be fetched manually."